### ISSUE TITLE
Shebang in script results in false error (spaced-comment)

### ISSUE THREAD

**Issue by necolas** [babel/babel-eslint#240]
Input:

```js
#!/usr/bin/env node
```

Result:

```bash
  1:1  error  Expected space or tab after "//" in comment  spaced-comment
```

**Comment by dasilvacontin** [comment]
Dup of https://github.com/babel/babel-eslint/issues/163.
